yb-voyager/cmd/root.go

```diff
@@ -65,12 +64,27 @@ Refer to docs (https://docs.yugabyte.com/preview/migrate/) for more details like
 	},
 
 	PersistentPostRun: func(cmd *cobra.Command, args []string) {
-		if exportDir != "" && utils.FileOrFolderExists(exportDir) && cmd.Use != "version" && cmd.Use != "status" && cmd.Use != "initiate" && !(cmd.Use == "setup" && cmd.Parent().Use == "fall-forward") {
+		if shouldLock(cmd) {
 			unlockExportDir()
 		}
 	},
 }
 
+func shouldLock(cmd *cobra.Command) bool {
+	if cmd.Use == "version" || cmd.Use == "status" {
+		return false
+	}
+	if cmd.Use == "initiate" && cmd.Parent().Use == "cutover" {
+		return false
+	}
+	if cmd.Parent().Use == "fall-forward" {
```

Comment:
also there are chances that there is no parent command, in that case this might result in `nil pointer exception` here